src/FacetedSearchContainer.js

```diff
@@ -12,16 +12,24 @@ class FacetedSearchContainer extends React.Component {
     super(props)
 
     this.state = {
-      facets:
+        facets:
         _(props.results)
           .flatMap(`facets`)
           .compact()   // lodash will emit undefined if `facets` doesn’t exist :/
           .uniqWith(_.isEqual)
           .map((facet) => ({...facet, disabled: false}))
           .value(),
-      selectedFacets: {}  // TODO (?) Build initial state of checked filters from props if wrapped in React Router
+        selectedFacets : this.props.selectedSpecies.length!==0 ?
```

Comment:
Remember that any new prop should be added to the component’s `propTypes`.

Reply:
Also, there’s no need to rely on a new prop to hide the species group. You could create an array of facets shared by all results and pass it to the sidebar, where checkbox facet groups can hide it. What you can add then is a prop to, maybe, explicitly hide such facets.

Additionally, this component tries to be general enough so that we (and others) can use it anywhere where search results with facets are needed. You can see this in the [README file](https://github.com/ebi-gene-expression-group/scxa-faceted-search-results/blob/master/README.md), and you won’t see any explicit references to species, marker genes, etc.